api/ingress/v1alpha1/ngrok_common.go

```diff
@@ -431,6 +431,7 @@ func (amazon *EndpointOAuthAmazon) ToNgrok(clientSecret *string) *ngrok.Endpoint
 	return mod
 }
 
+type EndpointTrafficPolicy json.RawMessage
```

Comment:
We should define this in a new file `api/ngrok/v1alpha1/common.go`. This way as we can onboard the new endpoints content there as-well.

Reply:
We may want to revisit this in the plumbing stage for `ngrok/v1alpha1` kinds, but right now, colocation next to `EnpointPolicy` made sense to us in this diff because we are modifying mostly the `api/ingress` controller to post schemaless json.

Reply:
cool, in that case this can wait for the pluming step. The reason why I'm interested in removing this concept from the `api/ingress` is because I wish to separate all ngrok concepts from ingress and gateway. This way we can move towards creating `gateway` and `ingress` changes with minimal `ngrok` concept leaks